python/cudf/cudf/_lib/copying.pyx

```diff
@@ -265,3 +270,49 @@ def copy_range(out_col, in_col, int out_begin, int out_end,
     free_column(c_out_col)
 
     return out_col
+
+
+def scatter_to_tables(source, maps):
```

Comment:
Should this be `scatter_to_frames` instead?

Reply:
```suggestion
def scatter_to_frames(source, maps):
```